bagit.py

```diff
@@ -787,14 +787,17 @@ def _make_tagmanifest_file(alg, bag_dir):
             tagmanifest.write('%s %s\n' % (digest, filename))
 
 def _find_tag_files(bag_dir):
-    for dir_name, _, filenames in os.walk(bag_dir):
-        if not re.match(r'.*data$', dir_name):
-            for filename in filenames:
-                if filename.startswith('tagmanifest-'):
-                    continue
-                #remove everything up to the bag_dir directory
-                p = join(dir_name, filename)
-                yield os.path.relpath(p, bag_dir)
+    for dir in os.listdir(bag_dir):
```

Comment:
`dir` is a builtin Python function - it probably won't break anything but I prefer not use redefine those names where possible.